resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+restartfile=<%= EZBake::Config[:restart_file] %>
+timeout="300"
```

Comment:
should timeout here use the start-timeout ezbake var instead of being hardcoded to 300?